hasher-matcher-actioner/Dockerfile

```diff
@@ -14,3 +27,5 @@ ARG DEPS_PATH
 COPY --from=build $DEPS_PATH $DEPS_PATH
 ENV PYTHONPATH $DEPS_PATH
 COPY hmalib ./hmalib
+
+RUN python3 ./hmalib/lambdas/fetcher.py
```

Comment:
This seems off. `fetcher.py` does not have an `if __name == "__main__":` block.

Did you accidentally check this in?

Reply:
Yes. This was how I was testing it could run correctly. Removing